audio_service/lib/audio_service.dart

```diff
@@ -262,26 +261,20 @@ class PlaybackState {
   Duration get position {
     if (playing && processingState == AudioProcessingState.ready) {
       return Duration(
-        milliseconds: (updatePosition.inMilliseconds +
-                speed *
-                    (clock.now().millisecondsSinceEpoch -
-                        updateTime.millisecondsSinceEpoch))
-            .toInt(),
+        milliseconds:
+            (updatePosition.inMilliseconds + speed * (clock.now().millisecondsSinceEpoch - updateTime.millisecondsSinceEpoch)).toInt(),
```

Comment:
Likewise here, please leave the indentation unmodified by using either `dart format` or the same indentation rules as `dart format`.